.github/workflows/ci.yaml

```diff
@@ -1,10 +1,7 @@
 on:
-  push:
-    branches:
-      - main
   pull_request:
     branches:
-      - main
+      - feature/**
```

Comment:
- re-running the github CI workflow in this PR to double-check everything is functional after this hotfix
- expecting to kickoff the CRAN release soon